project/Makefile.jinja

```diff
@@ -7,14 +7,16 @@ args = $(foreach a,$($(subst -,_,$1)_args),$(if $(value $a),$a="$($a)"))
 check_code_quality_args = files
 docs_serve_args = host port
 release_args = version
-test_args = cleancov match
+test_args = match
 
 BASIC_DUTIES = \
 	changelog \
 	clean \
 	coverage \
 	docs \
+	[%- if repository_provider == "github.com" %]
 	docs-deploy \
+	[%- endif %]
```

Comment:
This action is not needed for GitLab. Docs are built and deployed from CI, using the `docs` action.